### PR TITLE
CMPDC- 934: Moves Select component from sling-web

### PR DESCRIPTION
**Pull request by joselitojunior1**
### Description
This PR brings `Select` component from the sling-web repository.

### How has this been tested?
By running `npm install && npm rebuild && npm test && npm run spec`. Also by starting the components with `npm start sling-web-component-select`

⚠️ This is a subtask from CMPDC-905

### CODE REVIEW

File: packages/sling-web-component-select/public/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      name: 'Option 7',
+      id: 7,
+    },
+    {
+      name: 'Option 8',
+      id: 8,
+    },
+  ];
+
+  const $Select = document.querySelector('sling-select');
```
Comment (leofavre) [inline packages/sling-web-component-select/public/index.html]: We prefer `$lowercase`.

Comment (joselitojunior1) [reply packages/sling-web-component-select/public/index.html]: Thats make sense. Changing it right now

**Pull request merged**